README.md

```diff
@@ -24,13 +19,19 @@ This is found at `./static-site/`.
 See [static-site/README.md](./static-site/README.md) for instructions on how to add documentation and develop.
 This is built using Gatsby during deployment and served via `./server.js`.
 
-
+---
 ## Auspice
-Auspice is the software used to visualise & interact with phylogenomic data.
-The auspice customisations are found in `./auspice/client/`, with the server code for handling auspice-derived requests for datasets etcetera in `./auspice/server`.
+[Auspice](https://github.com/nextstrain/auspice) is the web-facing (e.g. "client") software used to visualise & interact with phylogenomic data.
+When nextstrain.org is built, it uses a JavaScript bundle from a previously installed version of auspice.
```

Comment:
Could you clarify this a little bit? 

do you mean while running `./set-up.sh`, or while running `node server.js`? The former will install a fresh copy auspice and build the client (i.e. the bundle) using nextstrain.org provided customisations. The latter isn't building anything, and expects a built client to exist (which it'll serve).